doc/_data/schemas/State/Grippers.json

```diff
@@ -3,22 +3,25 @@
   [
     {
       "title": "mc_control::fsm::Grippers",
-      "description": "Controllers the gripper of a robot",
+      "description": "Controls the grippers of a robot",
       "type": "object",
       "properties":
       {
         "robot": { "type": "string",
                    "title": "Robot this is applied to",
                    "description": "If ommited, the state applies to the main robot"
         },
-        "grippers": { "type": "object",
-                      "title": "Grippers' target",
-                      "properties":
-                      {
-                        "*": { "$ref": "/../../common/fsm_gripper.json" }
-                      },
-                      "description": "Keys are interpreted as grippers' name"
-        }
+        "grippers":
+        {
+          "type": "array",
+          "description": "Map of gripper name -> gripper configuration",
+          "items":
+          [
+            { "type": "string", "description": "Gripper name" },
+            { "$ref": "/../../mc_control/generic_gripper.json" }
```

Comment:
@arntanguy you forgot to include this file in your PR, can you push it to master? This breaks the doc generation

Also, can you keep this entry as an object type? The code does not expect an array